### ISSUE TITLE
Clusters having one data point

### ISSUE THREAD

**Issue by khadijakhaldi** [scikit-learn-contrib/hdbscan#362]
Hello,
Please at the first training steps of my model, my clusters may have one data point. Can HDBSCAN work in this case as it neglects points that are not assigned to any cluster?

**Comment by lmcinnes** [comment]
I don't think HDBSCAN would be well suited to that kind of problem. Ultimately density based techniques require a large enough amount of data to get traction (via density estimates). If clusters can have a single observation it is unlikley that density based techniques will manage to detect them. Sorry.

**Comment by khadijakhaldi** [comment]
thank you for your response!  I tried manually to assign the points considered as noise by HDBSCAN as clusters having one data point, in order that I can use them in my next iterations. Do you think it's ok to do so ? (I tried it and it is giving me better results for my case)

**Comment by lmcinnes** [comment]
Yes -- taking noise to to be individual clusters of single data points is a
valid interpretation, and the correct one for various downstream tasks.
Since it is working better for you I suspect that your downstream task is
exactly one of those cases.
